scripts/validate-notebooks.ps1

```diff
@@ -83,16 +83,24 @@ function Validate {
             </configuration>
         " | Out-File ./NuGet.Config -Encoding utf8
 
+		# Specifying tasks that need to be excluded because of various reasons :
```

Comment:
I would replace this comment with a link to the corresponding section of CONTRIBUTING.md - this way we won't have to maintain the explanations of the tags in two places. We'll still have to keep the *list* of tags in both places (in the script for actual validation and in CONTRIBUTING.md for explanations), but that's a lot easier.

Reply:
Done